datasets/fws-nwi/dataset.yaml

```diff
@@ -4,20 +4,27 @@ args:
   - registry
 code:
   src: ${{ local.path(./fws_nwi.py) }}
-  requirements: ${{ local.path(./requirements.txt) }}
+  # requirements: ${{ local.path(./requirements.txt) }}
 environment:
   AZURE_TENANT_ID: ${{ secrets.task-tenant-id }}
   AZURE_CLIENT_ID: ${{ secrets.task-client-id }}
   AZURE_CLIENT_SECRET: ${{ secrets.task-client-secret }}
+
+task_config:
+  fws-nwi:
+    create-items:
+      tags:
+        batch_pool_id: high_memory_pool
```

Comment:
A handful of states were failing on the regular task pool. They all succeeded with the high-memory pool.